actionpack/lib/action_dispatch/routing/route_set.rb

```diff
@@ -95,9 +96,23 @@ def clear!
             @url_helpers_module.send  :undef_method, helper
           end
 
+          @custom_helpers.each do |helper|
+            path_name = :"#{helper}_path"
+            url_name = :"#{helper}_url"
+
+            if @path_helpers_module.method_defined?(path_name)
+              @path_helpers_module.send :undef_method, path_name
```

Comment:
It was changed in 0088b08dcaf16176c8f9364d1d786f0c3728d369 - @tenderlove any reason `undef_method` was used instead of `removed_method` ?